libr/core/cmd_debug.c

```diff
@@ -5203,9 +5203,8 @@ static int cmd_debug(void *data, const char *input) {
 #define P r_cons_printf
 #define PS(X, Y) {escaped_str = r_str_escape (Y);r_cons_printf(X, escaped_str);free(escaped_str);}
 				if (rdi) {
-					const char *s = r_signal_to_string (core->dbg->reason.signum);
 					P ("type=%s\n", r_debug_reason_to_string (core->dbg->reason.type));
-					P ("signal=%s\n", r_str_get_fail (s, "none"));
+					P ("signal=%s\n", r_signal_to_human (core->dbg->reason.signum));
```

Comment:
Keep the signal as the old tostring and add sigstr for the human friendly string. So its backward compatible

Reply:
@trufae do you mean P ("sigstr=%s\n", r_signal_to_human (core->dbg->reason.signum));

Or do you mean what I did in the last push -f (a code that include `P ("stopreason=%s\n", r_signal_to_human (core->dbg->reason.signum));`)?